### PR TITLE
BUG/ENH: ACE Meta data and unused time dimensions

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses #186

Creates a custom load routine for ACE data products.  Data is loaded as xarray, then extra dimensions (labels for string time values) are dropped.
- epam and swepam converted to pandas DataFrame after extra dimensions are dropped
- Updates label types to match pysat standards `(float, int, str)`
- Changes directly applied since ACE instruments are not released in a previous version
- Fixed a bug where the ACE clean routine was trying to apply fill vals to coords

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Using the workflow in #186 and checking the metadata

## Test Configuration
* Operating system: Monterrey
* Version number: python 3.10.9

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes
- [x] Update zenodo.json file for new code contributors

### CODE REVIEW

**Comment by jklenzing** [comment]
warnings down from 4505 to 1277

**Comment by jklenzing** [comment]
Still a few to fix, all in the mag_l2 key 1hr instrument.
```
import pysat
mag = pysat.Instrument('ace', 'mag_l2', tag='key', inst_id='1hr', use_header=True)
mag.load(2020, 1)
```
All three variables throw the 'meta dropped for fill' warning when run with the clean routine, but are fine info clean_level is set to None.  Inspecting the metadata, it looks like everything is transferred correctly.

File: pysatNASA/instruments/methods/ace.py [hunk 1]
```diff
@@ -34,6 +36,59 @@
...
+                          meta_translation=meta_translation,
+                          drop_dims=['dim_empty', 'dim0', 'unit_time'],
+                          use_cdflib=True)
+
+    if to_pandas:
+        try:
+            data = data.to_pandas()
+        except AttributeError:
+            # xarray 0.16 support required for operational server
+            data = data.to_dataframe()
```
Comment (aburrell) [inline pysatNASA/instruments/methods/ace.py:87]:
```suggestion
        if hasattr(data, 'to_pandas'):
            data = data.to_pandas()
        else:
            # xarray 0.16 support required for operational server
            data = data.to_dataframe()
```

File: pysatNASA/instruments/methods/ace.py [hunk 2]
```diff
@@ -43,8 +98,15 @@ def clean(self):
 
     """
 
+    # Get a list of coords for the data
+    if self.pandas_format:
+        coords = [self.data.index.name]
+    else:
+        coords = [key for key in self.data.coords]
```
Comment (aburrell) [inline pysatNASA/instruments/methods/ace.py:105]:
```suggestion
        coords = [key for key in self.data.coords.keys()]
```

Stylistically, we haven't been doing the implied key output.

**Pull request merged**